#!/usr/bin/env python
PACKAGE = "aqronos_fft_record"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

para_datatype_enum = gen.enum([ gen.const("adc_data",      int_t, 0, "A small constant"),
                            gen.const("ff_data",     int_t, 1, "A medium constant")], "para stata")

gen.add("record_datatype", int_t, 0, "curt record data type param", 0, 0, 1, edit_method=para_datatype_enum)

gen.add("write_enable", bool_t, 0, "udp data write to file function", False)
gen.add("file_name", str_t, 0, "udp pkt file name.", "fft_data")

exit(gen.generate(PACKAGE, PACKAGE, "udppkt_write"))
